tests/integration/140.command.sites.test.js

```diff
@@ -37,8 +37,11 @@ const { createSitesFromTemplateCommand, fetchTemplates } = require('../../src/co
 /* eslint-enable import/order */
 const { withMockApi } = require('./utils/mock-api')
 
-test.afterEach(() => {
-  sinon.restore()
+test.beforeEach(() => {
```

Comment:
These tests were flaky due to not running in `serial` and not clearing the fakes before each test.